### PR TITLE
[INLONG-8199][Manager] Supports get brief information of inlong stream

### LINKED ISSUE #8199: [Improve][Manager] Supports get brief information of inlong stream
### Description

Supports get brief information of inlong steam.
Now` /stream/get` pulls all the source and sink information under the stream, however, users often do not need to use these information, when there are too many sources, this interface will be very time-consuming, so add an interface that only gets stream information.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8199 

### Motivation

Supports get brief information of inlong stream.

### Modifications

Supports get brief information of inlong stream.

### CODE REVIEW

**Review by vernedeng — COMMENTED** [review]
LGTM

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java [hunk 1]
```diff
@@ -231,7 +231,8 @@ private void evictClusterNode(HeartbeatMsg heartbeat) {
         // heartbeatInterval() is the reporting interval of cluster nodes, multiplied by two to prevent network
         // fluctuations
         ComponentHeartbeatEntity componentHeartbeatEntity = componentHeartbeatMapper.selectTimeOutHeartBeat(
-                componentHeartbeat.getComponentType(), componentHeartbeat.getIp(), heartbeatInterval() * 2L);
+                componentHeartbeat.getComponentType(), componentHeartbeat.getIp(),
+                heartbeatInterval() * heartbeatIntervalFactor);
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java:235]: It is not relative with PR description

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java [hunk 2]
```diff
@@ -277,6 +277,32 @@ public InlongStreamInfo get(String groupId, String streamId, UserInfo opInfo) {
         return streamInfo;
     }
 
+    @Override
+    public InlongStreamBriefInfo getBrief(String groupId, String streamId, String operator) {
+        InlongGroupEntity entity = groupMapper.selectByGroupId(groupId);
+        if (entity == null) {
+            throw new BusinessException(ErrorCodeEnum.GROUP_NOT_FOUND);
+        }
+        // only the person in charges can query
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java:286]: Add one blank line, to improve the code readability.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java]: Remove it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java [hunk 3]
```diff
@@ -277,6 +277,32 @@ public InlongStreamInfo get(String groupId, String streamId, UserInfo opInfo) {
...
+        userService.checkUser(entity.getInCharges(), operator,
+                ErrorCodeEnum.GROUP_PERMISSION_DENIED.getMessage());
+        // get stream information
+        InlongStreamEntity streamEntity = streamMapper.selectByIdentifier(groupId, streamId);
+        if (streamEntity == null) {
+            throw new BusinessException(ErrorCodeEnum.STREAM_NOT_FOUND);
+        }
+        InlongStreamBriefInfo streamInfo = CommonBeanUtils.copyProperties(streamEntity, InlongStreamBriefInfo::new);
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java:296]: Done.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**